src/main/java/fr/insalyon/citi/golo/runtime/FunctionCallSupport.java

```diff
@@ -108,11 +118,15 @@ public static Object fallback(FunctionCallSite callSite, Object[] args) throws T
       Method method = (Method) result;
       checkLocalFunctionCallFromSameModuleAugmentation(method, callerClass.getName());
       types = method.getParameterTypes();
-      if (method.isVarArgs() && isLastArgumentAnArray(types.length, args)) {
+      //TODO improve varargs support on named arguments. Matching the last param type + according argument
```

Comment:
I have a feeling that vararg arguments in presence of named arguments do not mix well... @yloiseau @Artpej what do you think?